src/World.cpp

```diff
@@ -11,20 +11,28 @@
 
 using namespace ldtk;
 
-World::World(const nlohmann::json& j, Project* p, bool external_levels) :
+ldtk::World::World(const nlohmann::json& j, Project* p) :
```

Comment:
Why is the namespace added here ? Is it by mistake ?

Reply:
This was a leftover artefact from my IDE's auto-generation. I removed most of them, but it seems this one sneaked through.

Reply:
Okay no problem. 

I don't know how familiar you are with git, but it would be nice if you could make the last commit "[Remove unnecessary namespace specifiers](https://github.com/Madour/LDtkLoader/pull/23/commits/a209b8b90d11ccc529da0424f831f15d2bf1a0c3)" a fixup to the commit "[Implement support for custom file loaders](https://github.com/Madour/LDtkLoader/pull/23/commits/134cf90db5ada6885f2e448cdc5f3c8533cbbbb3)" that introduced the extra namespace by accident.

Also, moving "[Fix incorrect header](https://github.com/Madour/LDtkLoader/pull/23/commits/03c7fcd3f9264e093e723384cfa51721a6c67328)" to be the first commit of this PR would be also nice.